cf-serverd/server_common.c

```diff
@@ -1003,20 +1003,10 @@ int StatFile(ServerConnectionState *conn, char *sendbuffer, char *ofilename)
 
         cfst.cf_readlink = linkbuf;
     }
-#endif /* !__MINGW32__ */
```

Comment:
Moving this means skipping the following code on MINGW, which we didn't previously.
How is that right ?